examples/go/with_proto/run_configurations/test_macro_expansion.xml

```diff
@@ -0,0 +1 @@
+<configuration name="Bazel test go:go_test" type="BlazeCommandRunConfigurationType" factoryName="Bazel Command" nameIsGenerated="true"><blaze-settings handler-id="BlazeGoRunConfigurationHandlerProvider" blaze-command="test" kind="go_test"><blaze-target>//go:go_test</blaze-target><blaze-user-flag>--test_env=WORKSPACE_ROOT=$WorkspaceRoot$</blaze-user-flag><blaze-user-flag>--test_env=FILE_NAME=$FileName$</blaze-user-flag></blaze-settings><method v="2"><option name="Blaze.BeforeRunTask" enabled="true" /></method></configuration>
```

Comment:
I think we can add (--test_filter=^TestFlagMacros$ --test_env=GO_TEST_WRAP_TESTV=1) blaze flags for the test results to be viewed correctly in the IDE.

Reply:
Done! Thanks for the suggestion.